generate/avro_helpers.go

```diff
@@ -337,6 +337,9 @@ func (fm *avroFieldMapper) generateStructValue(name, prefix, typeName string, ge
 		} else {
 			structDef = mappedStruct.name
 		}
+		if _, ok := aType.Elt.(*ast.StarExpr); ok {
```

Comment:
Am I missing anything or is this the only non-generated change?

Reply:
I added the contributors array to the simple schema as a test case for this, otherwise yes the rest are generated changes.

Reply:
👍